external_parser/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ if(WIN32 AND (STATIC_LINK_BINARY_PARSER))
 endif()
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/Modules/")
-find_package(flatbuffers REQUIRED)
+find_package(Flatbuffers REQUIRED)
```

Comment:
is there something breaking to trigger this change?

Reply:
Yea somethings up with the flatbuffers dependency for the binary parser, dont think this fixes it

Reply:
https://dev.azure.com/vowpalwabbit/Vowpal%20Wabbit/_build/results?buildId=54163&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86&l=28